app/controllers/dashboard.js

```diff
@@ -14,19 +14,17 @@
  * limitations under the License.
  */
 
-import Ember from "ember";
+const Ember = require('ember');
 
-var DashboardController = Ember.ObjectController.extend({
+const DashboardController = Ember.ObjectController.extend({
   chartColorCache: [],
   reservedCollections: ['_outgoingPushMessages', '_outgoingEmailMessages', '_blLogs'],
-  userAccessibleCollections: Ember.computed.filter('model.collectionStats', function(collection) {
+  userAccessibleCollections: Ember.computed.filter('model.collectionStats', function (collection) {
```

Comment:
Given this rule (https://github.com/Kinvey/style-guides/blob/master/backend/es6.md#arrows--use-them), shouldn't this be 
```suggestion
  userAccessibleCollections: Ember.computed.filter('model.collectionStats', (collection) => {
```

Same for other instances below (for example line 27).

Reply:
Nope, because using the arrow function will cause lose of `this` context used at the next line.